crates/holochain/src/conductor/conductor/builder.rs

```diff
@@ -55,8 +54,33 @@ pub struct ConductorBuilder {
 
 impl ConductorBuilder {
     /// Default ConductorBuilder.
+    #[allow(clippy::new_without_default)]
     pub fn new() -> Self {
-        Self::default()
+        static CRYPTO: std::sync::Once = std::sync::Once::new();
+        CRYPTO.call_once(|| {
+            if rustls::crypto::aws_lc_rs::default_provider()
+                .install_default()
+                .is_ok()
+            {
+                tracing::warn!("Defaulting to aws-lc as the rustls crypto provider. Please set this in your binary before invoking the ConductorBuilder. https://docs.rs/rustls/latest/rustls/crypto/struct.CryptoProvider.html#using-the-per-process-default-cryptoprovider");
```

Comment:
If the installation here succeeds, what should be set in a binary?

Reply:
No, this is warning that it should be set in the binary crate, as opposed to being defaulted here.

Reply:
Ah yes, got it.